### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by selwiak**

### CODE REVIEW

File: mystring.h [hunk 1]
```diff
@@ -14,5 +14,5 @@ class String {
     ~String();
     const char* toString() const;
     friend std::ostream & operator<< (std::ostream& os, const String &s);
-    friend unsigned int stringLength(const String mystring);
+    friend unsigned int stringLength(char* const str);
```
Comment (dmytro-dubovyk) [inline mystring.h]: it's better to use it as a class member, just like "length()"

Comment (dmytro-dubovyk) [reply mystring.h]: the idea is that noone has access to the internal string apart from the class itself

File: mystring.h [hunk 2]
```diff
@@ -13,4 +14,5 @@ class String {
     ~String();
     const char* toString() const;
     friend std::ostream & operator<< (std::ostream& os, const String &s);
+    friend unsigned int stringLength(const String mystring);
```
Comment (dmytro-dubovyk) [inline mystring.h]: same here, please look at my previous comment

**Review by dmytro-dubovyk — CHANGES_REQUESTED** [review]

File: mystring.h (hunk 1 above)
Comment (selwiak) [reply mystring.h]: removed friend function and added class member "length()"

**Pull request merged**